scipy/stats/_stats_py.py

```diff
@@ -8107,26 +8107,27 @@ def brunnermunzel(x, y, alternative="two-sided", distribution="t",
 
 def combine_pvalues(pvalues, method='fisher', weights=None):
     """
-    Combine p-values from independent tests bearing upon the same hypothesis.
+    Combine p-values from independent tests that bear upon the same hypothesis
+    and are based on continuous distributions.
 
     Parameters
     ----------
     pvalues : array_like, 1-D
-        Array of p-values assumed to come from independent tests.
-    method : {'fisher', 'pearson', 'tippett', 'stouffer',
-              'mudholkar_george'}, optional
+        Array of p-values assumed to come from independent tests based on
+        continuous distributions.
+    method : {'fisher', 'pearson', 'tippett', 'stouffer', 'mudholkar_george'}, optional
```

Comment:
Please revert the change on `method` as the formatting is correct. Another alternative would be to remove `optional` since the first parameter is the default in numpydoc terms.
```suggestion
    method : {'fisher', 'pearson', 'tippett', 'stouffer', 'mudholkar_george'}
```

Reply:
> Please revert the change on `method` as the formatting is correct.

Well, it doesn’t [get properly rendered](https://docs.scipy.org/doc/scipy/reference/generated/scipy.stats.combine_pvalues.html). Removing the `optional` is fine by me though.

Reply:
Indeed, I will have a look at similar cases. It should work or there is a small tweak I don't see here.

Reply:
I am sure it was working before, I see other places where this is broken as well. Thanks for noticing this.